### PR TITLE
[Cart] Remove an entire project from the cart.

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

remove-project

Update the remove button to reveal a dropdown menu where the user can select whether they want to remove the sample or the entire project from the cart.

## Related issue
Related to #192 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~~
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

**Comment by apetkau** [comment]
Hey @joshsadam can you change this to merge into the `cart/_base` branch?

**Comment by joshsadam** [comment]
Sorry @apetkau it was all merged up, I had this merge pointed to development instead of base.  Thanks for noticing the error.

File: src/main/webapp/resources/js/pages/cart/components/SampleRenderer.jsx [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-import { actions } from "../../../redux/reducers/cart";
-import { Button, Col, Icon, Row, Tooltip } from "antd";
+import { Button, Col, Dropdown, Icon, Menu, Row } from "antd";
 import { COLOURS } from "../../../styles";
 
-export default class SampleRenderer extends React.Component {
+const DeleteMenu = ({ removeSample, removeProject }) => (
+  <Menu>
+    <Menu.Item>
+      <div onClick={removeSample}>Remove Sample</div>
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/cart/components/SampleRenderer.jsx]: Updated in a1c0016bb0461ef506ff50913cbf39f4567465d7.

Not sure about your above comment.  I tried to reproduce it, and I get a notification that that sample is in that cart.  Could you show me?

[the remark this answers is not in the document]
Comment (apetkau) [reply src/main/webapp/resources/js/pages/cart/components/SampleRenderer.jsx]: Awesome, thanks. The internationalization looks good.

[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/cart/components/SampleRenderer.jsx]: Thank @apetkau fixed that notification issue in ee84dfaef665bc82cec9429d9c88d539574c0476

**Pull request merged**